### PR TITLE
Displays team and player names on match screen.

### PR DESCRIPTION
**Pull request by jonatanlandin**

### CODE REVIEW

**Review by cajoho99 — APPROVED** [review]

File: packages/backend/src/modules/tournament/tournament.ts [hunk 1]
```diff
@@ -22,11 +22,11 @@ export default class Tournament {
 	})
 	matches: Match[];
 
-	@Field(() => Match, {
+	@Field(() => [Match], {
 		description: 'References all the matches that are connected to the tournament',
 	})
 	// TODO: needs to be | undefined else everything be bork
-	contestedMatches?: Match[];
+	contestedMatches?: Match[] | undefined;
```
Comment (rosengrenen) [inline packages/backend/src/modules/tournament/tournament.ts]: ????????????????? why undefined?

File: packages/frontend/src/screens/app/screens/tournaments/screens/tournament/screens/match/match-view.tsx [hunk 2]
```diff
@@ -23,6 +27,7 @@ const MatchView: React.FC<Props> = ({ reportWin, reportContested, match, loading
 	if (match.winner) {
 		return (
 			<div>
+				<MatchViewItem key={match.id} match={match} />
```
Comment (rosengrenen) [inline packages/frontend/src/screens/app/screens/tournaments/screens/tournament/screens/match/match-view.tsx]: remove key

File: packages/frontend/src/screens/app/screens/tournaments/screens/tournament/screens/match/match-view.tsx [hunk 3]
```diff
@@ -37,6 +42,7 @@ const MatchView: React.FC<Props> = ({ reportWin, reportContested, match, loading
 
 	return (
 		<div>
+			<MatchViewItem key={match.id} match={match} />
```
Comment (rosengrenen) [inline packages/frontend/src/screens/app/screens/tournaments/screens/tournament/screens/match/match-view.tsx]: remove key

**Review by rosengrenen — CHANGES_REQUESTED** [review]

File: packages/frontend/src/screens/app/screens/tournaments/screens/tournament/screens/manage-tournament/manage-tournament-data.tsx [hunk 4]
```diff
@@ -41,9 +41,9 @@ const ManageTournamentData: React.FC<Props> = ({ tournamentId }) => {
 		},
 	);
 
-	if (loading) return <div>Ladd för fan</div>;
+	if (loading) return <div>Loading...</div>;
```
Comment (rosengrenen) [inline packages/frontend/src/screens/app/screens/tournaments/screens/tournament/screens/manage-tournament/manage-tournament-data.tsx:44]: ONE LINE IF STATEMENTS? SMORC

File: packages/frontend/src/screens/app/screens/tournaments/screens/tournament/screens/manage-tournament/manage-tournament-data.tsx [hunk 5]
```diff
@@ -41,9 +41,9 @@ const ManageTournamentData: React.FC<Props> = ({ tournamentId }) => {
...
 
-	if (error) return <div>stfu bith is errOr</div>;
+	if (error) return <div>Error</div>;
```
Comment (rosengrenen) [inline packages/frontend/src/screens/app/screens/tournaments/screens/tournament/screens/manage-tournament/manage-tournament-data.tsx:46]: this too smorc

**Review by rosengrenen — DISMISSED** [review]

**Review by Noxemia — APPROVED** [review]

**Review by rosengrenen — APPROVED** [review]

**Pull request merged**